### ISSUE TITLE
Error when playing "Ellie Goulding - Lights (Fytch Remix)" (B3WMlT7wXg0)

### ISSUE THREAD

**Issue by natumbri** [natumbri/mopidy-youtube#132]
@BrainStone reports that, even after reinstalling from develop with the changes made to fix Issue #122 , he has difficulty playing "Ellie Goulding - Lights (Fytch Remix)" (B3WMlT7wXg0)

The log he gets is:
```
INFO     [Thread-64] mopidy_youtube session.get triggered: list_videos (experimental)
INFO     [YouTubeBackend-7] mopidy_youtube youtube LibraryProvider.lookup "youtube:video/Ellie Goulding - Lights (Fytch Remix).B3WMlT7wXg0"
ERROR    [Core-10] mopidy.core.library YouTubeBackend backend caused an exception.
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/dist-packages/mopidy/core/library.py", line 17, in _backend_error_handling
    yield
  File "/usr/local/lib/python3.7/dist-packages/mopidy/core/library.py", line 217, in lookup
    result = future.get()
  File "/usr/local/lib/python3.7/dist-packages/pykka/_threading.py", line 45, in get
    _compat.reraise(*self._data['exc_info'])
  File "/usr/local/lib/python3.7/dist-packages/pykka/_compat/__init__.py", line 29, in reraise
    raise value
  File "/usr/local/lib/python3.7/dist-packages/pykka/_actor.py", line 193, in _actor_loop
    response = self._handle_receive(envelope.message)
  File "/usr/local/lib/python3.7/dist-packages/pykka/_actor.py", line 299, in _handle_receive
    return callee(*message.args, **message.kwargs)
  File "/usr/local/lib/python3.7/dist-packages/mopidy_youtube/backend.py", line 196, in lookup
    name=video.title.get().replace(";", ""),
AttributeError: 'NoneType' object has no attribute 'replace'
[youtube] B3WMlT7wXg0: Downloading webpage
```

**Comment by natumbri** [comment]
@BrainStone, unfortunately(?), this track plays just fine for me, so I can't really test the issue.  It would be really helpful to see what YouTube is serving up to you.

Do you think you could record the text of the result of the session.get() that is executed when you try to play that track?

Something like the following, after line 25 in youtube_bs4api.py should do the trick:
```
        filename = "B3WMlT7wXg0.html"
        with open(filename, 'w') as f:
          f.write(result.text)
```
Add the track; generate the html; then upload `B3WMlT7wXg0.html` to a gist or similar for me to review.

Cheers,
Nik

**Comment by BrainStone** [comment]
Well...  
I checked the installation again and noticed something there:
`  Using cached https://github.com/natumbri/mopidy-youtube/archive/develop.zip`

Well that solves that mystery (and issue).

**Comment by natumbri** [comment]
Great, glad it is working for you now. And good to know the changes to fix #122 are effective for other videos.

I'll close this issue now.
